frame/election-provider-multi-phase/src/mock.rs

```diff
@@ -528,6 +535,10 @@ impl ExtBuilder {
 		<UnsignedPhase>::set(unsigned);
 		self
 	}
+	pub fn throttling_blocks(self, bn: BlockNumber) -> Self {
```

Comment:
```suggestion
	pub fn min_electing_blocks(self, bn: BlockNumber) -> Self {
```

based on other suggestion.